src/NexusMods.App.UI/RightContent/FoundGamesViewModel.cs

```diff
@@ -33,7 +31,6 @@ private async Task ManageGame(GameInstallation installation)
     {
         var name = _loadoutRegistry.SuggestName(installation);
         var marker = await _loadoutRegistry.Manage(installation, name);
```

Comment:
What's the plan here, move the navigation up or will it still happen here?

Reply:
Currently the navigation here is problematic because it only happens after the game has bee completely indexed and backed up, while ideally it would switch immediately to a new loadout with the GameFiles mod entry loading.

Just a note for later